
def vartest(var1):
    var1=1

var1=var1+1
#함수에서 사용하는 지역변수를 외부에서 사용하려고 했기때문에 에러발생
